frontend/components/App.js

```diff
@@ -188,10 +188,10 @@ export default class App extends React.Component {
                     >
                       <strong>{m.name}</strong>
                     </a>{" "}
-                    at{" "}
+                    on{" "}
                     {moment(m.scheduled_for)
                       .utc()
-                      .format("dddd, MMMM Do YYYY, h:mma")}{" "}
+                      .format("dddd, MMMM Do YYYY, [at] h:mma")}{" "}
```

Comment:
what does [at] do?

Reply:
It's an escape for Moment's formatting. `[something]` lets you insert strings at some point in the middle of it.